### PR TITLE
Feature/improve tales header

### PR DESCRIPTION
**Pull request by daria31v**
image link fixes

### CODE REVIEW

File: pages/kids/stories/[story].tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const StoriesContainer = ({ story, phrases }: StoriesProps): ReactNode => {
       <SEO
         title={`${title_current + ' - '}` + t(`seo.kids_page_storiesTitle.${getCountryVariant()}`)}
         description={t(`seo.kids_page_storiesDescription.${getCountryVariant()}`)}
-        image={`https://www.movapp.cz/kids/' + ${image_current_story} + '.jpg'`}
+        image={`'https://www.movapp.cz/kids/' + ${image_current_story} + '.jpg'`}
```
Comment (jejdacz) [inline pages/kids/stories/[story].tsx:46]: I think you don't need any ...
image={`https://www.movapp.cz/kids/${image_current_story}.jpg`}

Comment (daria31v) [reply pages/kids/stories/[story].tsx:46]: we have a dynamic parameter ${}, we should use ``
or are you talking about a different way of specifying the link?

Comment (met) [reply pages/kids/stories/[story].tsx:46]: @jejdacz Oh ,yes, you are right. My developing skills are old and obsolete ;)

Comment (met) [reply pages/kids/stories/[story].tsx:46]: Marek is using `` but github makes a mess with comments formating.

Comment (daria31v) [reply pages/kids/stories/[story].tsx:46]: Okay. Go it. I will change it tomorrow)

**Pull request merged**